rules/windows/network_connection/net_connection_win_script.yml

```diff
@@ -0,0 +1,24 @@
+title: Script Initiated Connection
+id: 08249dc0-a28d-4555-8ba5-9255a198e08c
+status: experimental
+description: Adversaries may use script to download malicious payloads.
+author: frack113
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/28d190330fe44de6ff4767fc400cc10fa7cd6540/atomics/T1105/T1105.md
+date: 2022/08/28
+logsource:
+    category: network_connection
+    product: windows
+detection:
+    selection:
+        Initiated: 'true'
+        Image|endswith: 
+            - '\wscript.exe'
+            - '\cscript.exe'
+    condition: selection
```

Comment:
Could we create another rule which is very similar to this one that check for a network connection of wscript/cscript to doesn't have a local IP as a target (10.x, 192.168.x, etc.)?